lib/core/registry.ex

```diff
@@ -204,7 +204,14 @@ defmodule TelemetryMetricsPrometheus.Core.Registry do
 
   defp register_metric(%Metrics.Distribution{} = metric, config) do
     case Distribution.register(metric, config.dist_table_id, self()) do
-      {:ok, handler_id} -> {:ok, {%{metric | buckets: metric.buckets ++ ["+Inf"]}, handler_id}}
+      {:ok, handler_id} ->
+        reporter_options = Keyword.update(
+          metric.reporter_options,
+          :buckets,
+          [],
```

Comment:
I reckon this could be improved, do you have any ideas @bryannaegele ?

Reply:
I don't think we need to provide a default as it's a required option, so we'll instead need to raise an exception during the validation phase. If you remove the empty list here, then this should be fine.

Reply:
This validation phase, probably would be best fit in `TelemetryMetricsPrometheus.Core.Distribution.register/3`? We have an error case there already, or should we put it in the `Registry.register_metric(%Metrics.Distribution{} = metric, config)` function instead?